src/mame/be/bebox_m.cpp

```diff
@@ -220,14 +217,9 @@ void bebox_state::bebox_crossproc_interrupts_w(offs_t offset, uint64_t data, uin
 			else
 				line = crossproc_map[i].active_high ? CLEAR_LINE : ASSERT_LINE;
 
-			if (LOG_INTERRUPTS)
-			{
-/*
-                logerror("bebox_crossproc_interrupts_w(): CPU #%d %s %s\n",
-                    crossproc_map[i].cpunum, line ? "Asserting" : "Clearing",
-                    (crossproc_map[i].inputline == PPC_INPUT_LINE_SMI) ? "SMI" : "TLBISYNC");
-                    */
-			}
+            LOGMASKED(LOG_INTERRUPTS, "bebox_crossproc_interrupts_w(): CPU #%d %s %s\n",
+				crossproc_map[i].cpunum, line ? "Asserting" : "Clearing",
+				(crossproc_map[i].inputline == 0/*PPC_INPUT_LINE_SMI*/) ? "SMI" : "TLBISYNC");
```

Comment:
Indent appears to be off on this block due to spaces/tabs after uncommenting it (the opening comment delimiter was at the left margin, so srcclean turned the entire indent into spaces).  It might come out OK again after srcclean, but it might not, too.

Reply:
This is what it looks like in my editor after running srcclean on the file. On the other hand, it's what it looked like in my editor before  running srcclean on the file. It might be that Github is doing something like treating tabs as 8-space tabs.

![bebox_m_indent](https://github.com/mamedev/mame/assets/1891617/a50091cf-ef9c-40e4-8738-52c1c8ec88e1)

Reply:
`git diff` does indicate some level of changes to the file after running srcclean, so I'll include that when addressing the feedback points below.

Reply:
srccleaned file pushed, marking resolved.